meson.build

```diff
@@ -194,6 +195,14 @@ if glib_dep.type_name() != 'internal'
   endif
 endif
 
+if dicom_dep.type_name() == 'internal' or cc.has_function(
+  'dcm_filehandle_get_frame_number',
+  dependencies : dicom_dep,
+)
+  # added in libdicom 1.2, needed for concatenation support
+  conf.set('HAVE_DICOM_GET_FRAME_NUMBER', 1)
```

Comment:
So on &lt; 1.2 we'd omit support for DICOM concatenations, right?  I think I'd rather just hard-require 1.2.0+.  In general, OpenSlide does not make any dependency optional that would affect format support, to prevent confusion about what is actually supported by a particular build of OpenSlide.

Reply:
OK, I'll remove the config var.